jobs/bump-lockfile.Jenkinsfile

```diff
@@ -162,6 +169,27 @@ delay_meta_merge: false
 EOF
                    """)
             gp.gangplankArchWrapper([spec: "spec.spec", arch: "aarch64"])
+        }, s390x: {
+            shwrap("""
+               cat <<'EOF' > spec.spec
+job:
+  strict: true
+  miniocfgfile: ""
+recipe:
+  git_ref: ${params.STREAM}
+  git_url: https://github.com/${repo}
+stages:
+- id: ExecOrder 1 Stage
+  execution_order: 1
+  description: Stage 1 execution base
+  build_artifacts: [base]
+  post_commands:
+    - cosa kola run --rerun --basic-qemu-scenarios
+    - rm -f builds/builds.json # https://github.com/coreos/coreos-assembler/issues/2317
+delay_meta_merge: false
```

Comment:
I have made it equal now by removing 'build_artifacts' and adding 'prep_commands'.

Reply:
> `kola testiso -SP --qemu-native-4k --scenarios iso-install --output-dir tmp/kola-metal4k`
there are currently no valid testing scenarios for metal4k images in case of s390x, so it is staged currently.
with this commit, the post commands are made equal to match aarch64

Reply:
yes, it is needed to build the metal4k image as it feeds to build the live image which otherwise fails and live build is a needed artifact.